Apps/WebClient/src/ClientApp/src/components/timeline/encounter.vue

```diff
@@ -65,7 +65,7 @@ export default class EncounterTimelineEntryComponent extends Vue {
                     <b-col>
                         <div class="d-flex flex-row-reverse">
                             <b-btn
-                                data-testid="detailsButton"
+                                data-testid="encounterDetailsButton"
```

Comment:
@sslaws Do you think if it is better to have a container div with data-testid="encouterContainer" above these encounter... as it may save us to make these testid shorter? This requires us to grab the encouterContainer then go deeper to get its elements in our Func. Tests Scripts

Reply:
I think we're looking for specific button press here so no need.  Long term we should look at optimizing tests and if you feel this test can be optimized please create a Task in DevOps.